Tests/LinuxMain.swift

```diff
@@ -185,6 +185,14 @@ extension MinimalNavViewTests {
     ("testNav_Screenshots", testNav_Screenshots)
   ]
 }
+extension NewBlogPostEmailTests {
+  static var allTests: [(String, (NewBlogPostEmailTests) -> () throws -> Void)] = [
+    ("testNewBlogPostEmail_NoAnnouncements_Subscriber", testNewBlogPostEmail_NoAnnouncements_Subscriber),
+    ("testNewBlogPostEmail_NoAnnouncements_NonSubscriber", testNewBlogPostEmail_NoAnnouncements_NonSubscriber),
+    ("testNewBlogPostEmail_Announcements_Subscriber", testNewBlogPostEmail_Announcements_Subscriber),
+    ("testNewBlogPostEmail_Announcements_NonSubscriber", testNewBlogPostEmail_Announcements_NonSubscriber)
```

Comment:
this stuff has nothing to do with the PR, I just forgot to generate linux tests in my last PR.